RecoParticleFlow/PFTracking/interface/PFCheckHitPattern.h

```diff
@@ -48,8 +48,8 @@ class PFCheckHitPattern {
   /// on track.
 
   PFTrackHitFullInfo 
-    analyze(edm::ESHandle<TrackerGeometry>, const reco::TrackBaseRef track, 
-	    const TransientVertex& vert);
+    analyze(const TrackerTopology* tkerTopo, const TrackerGeometry* tkerGeom,
+            const reco::TrackBaseRef track, const TransientVertex& vert);
```

Comment:
why the change over the `ESHandle` ? This is recommended somewhere so this is free cleaning ?